google/cloud/storage/quickstart/BUILD.bazel

```diff
@@ -32,7 +32,7 @@ cc_binary(
         "quickstart_grpc.cc",
     ],
     deps = [
-        "@google_cloud_cpp//:experimental-storage_grpc",
```

Comment:
I don't think this will build if we:

```sh
cd google/cloud/storage/quickstart
bazel build :quickstart_grpc
```

Can you try it and confirm? If it doesn't build, we should keep depending on the experimental target and leave a TODO to update to the non-experimental target when the next version of google-cloud-cpp is released and picked up in this repo by renovate-bot.

Reply:
Confirmed it did not work. Adding a todo.

Reply:
Added todo.